api/v1/models/Hacker.js

```diff
@@ -1,11 +1,13 @@
 /* jshint esversion: 6 */
 
-var registration = require('../utils/registration');
+var utils = require('../utils');
+var registration = utils.registration;
 var _ = require('lodash');
 
 var OMITTED_ATTRIBUTES = [];
```

Comment:
If you guys don't think there's a use, I can just remove it. I had it in place because I wasn't sure about registration parameters and whether all of them actually needed to be returned. Are these still being decided, or are they now finalized?